lily4178993/my-todos
# Implement My Todos UX

# Implement My Todos UX

## Description
This pull request introduces a user experience (UX) enhancement for the "My Todos" project. The following tasks have been completed:

## Checklist
- [X] Create a Todos application is using [Create React App](https://create-react-app.dev/)
- [X] Set-up linters:
  * Set-up ESLint
  * Set-up Stylelint
- [X] Create Components: `TodoForm`, `TodosList`, `TodoItem`
- [X] Add functionalities to components:
     * Implemented function for CRUD operation
- [X] Add sass Styles

## How to Test
To test this project:

1. Ensure you have cloned this branch and installed dependencies.
2. Run the development server:
   ```bash
   npm start
   ```
3. Open your web browser and access the application.

Please review my implementation and feel free to suggest any changes or improvements. Best regards.
